canvas_modules/common-canvas/__tests__/common-canvas/supernode-test.js

```diff
@@ -405,16 +405,18 @@ describe("Expand and Collapse Supernode Action", () => {
 		expect(canvasController.getNode(superNodeId).is_expanded).to.be.false;
 	});
 
-	it("Should set the is_expanded attribute correctly with undo and redo", () => {
-		canvasController.contextMenuHandler(expandCollapseDeconstructSourceObject);
-		canvasController.contextMenuActionHandler("expandSuperNodeInPlace");
-		expect(canvasController.getNode(superNodeId).is_expanded).to.be.true;
+	it("Should set the is_expanded attribute correctly with undo and redo", async() => {
+		setTimeout(() => {
```

Comment:
Is there a reason for using `setTimeout` here instead of `await`? Can you add a comment?

Reply:
Thanks for explaining @nmgokhale, I havent came across this issue before so I dont know of another way. Can you add a comment in the code for future note?